hosted/json-schemas/offering.schema.json

```diff
@@ -1,82 +1,152 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://tbdex.dev/offering.schema.json",
-  "definitions": {
-    "CurrencyDetails": {
+  "type": "object",
+  "properties": {
+    "additionalProperties": false,
+    "description": {
+      "type": "string",
+      "description": "Brief description of what is being offered."
+    },
+    "payin": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
         "currencyCode": {
           "type": "string",
           "description": "ISO 3166 currency code string"
         },
-        "minAmount": {
+        "min": {
           "$ref": "definitions.json#/definitions/decimalString",
           "description": "Minimum amount of currency that can be requested"
         },
-        "maxAmount": {
+        "max": {
           "$ref": "definitions.json#/definitions/decimalString",
           "description": "Maximum amount of currency that can be requested"
+        },
+        "methods": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "kind": {
+                "type": "string",
+                "description": "The type of payment method. e.g. BITCOIN_ADDRESS, DEBIT_CARD, etc."
+              },
+              "name": {
+                "type": "string",
+                "description": "Payment Method name. Expected to be rendered on screen."
+              },
+              "description": {
+                "type": "string",
+                "description": "Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. \"segwit addresses only\""
+              },
+              "group": {
+                "type": "string",
+                "description": "value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit"
```

Comment:
@ethan-tbd can you provide context?

Reply:
@angiejones for client applications, users have to select a desired payment method from the list of an offering's payment methods. but the current spec doesn't provide an easy/standard way to group these payment methods together (since `kind` is a unique identifier). by introducing `group` to the spec, client applications improve their ux by allowing users to select the type of payment method they are interested in before then selecting the exact payment method. (e.g. if an offering contains 6 payout methods but 3 of them are types of bank deposits and 3 of them are types of mobile money payouts, the user should decide what payout type they want before then selecting the exact bank or mobile money payout)

Reply:
would something like `availablePaymentMethods` be more descriptive?
also is there text in the spec that speaks to this?